sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -35,6 +36,11 @@ class DictVectorizer(TransformerMixin, BaseEstimator):
     a feature "f" that can take on the values "ham" and "spam" will become two
     features in the output, one signifying "f=ham", the other "f=spam".
 
+    If a feature value is a sequence or set of strings, this transformer
+    will iterate over the values and will similarly encode each string value
+    as a binary feature. Strings in the iterable are expected to be unique,
```

Comment:
Ok for counting all the occurrences of a string , but then I'm not encoding in a binary feature anymore, right?